### DISCUSSION TITLE
A3S -Atreseries | [Spain]

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#357] [Show and tell]
### Channel ID

Atreseries.es

### Stream URL

https://atres-live.atresmedia.com/84158ebc0c8885cccef1dbfe95c7cdd695718690_ES_1664474768_1664503508/h264/live/a3series/master.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by joaquinito2048** [comment]
please close this PR, this URL is signed and it appears not works.

**Comment by freearhey** [comment]
https://atres-live.atresmedia.com/84158ebc0c8885cccef1dbfe95c7cdd695718690_ES_1664474768_1664503508/h264/live/a3series/master.m3u8 (offline)
<img width="831" alt="image" src="https://user-images.githubusercontent.com/7253922/232235542-28ef40d3-07f6-4345-9521-68b0f1b42bde.png">
